.github/workflows/validate-and-test.yml

```diff
@@ -33,8 +33,8 @@ jobs:
           path: project/**/target
           key: build-target-${{ hashFiles('**/*.sbt', 'project/build.properties', 'project/**/*.scala') }}
 
-      - name: Autoformat
-        run: sbt +headerCreateAll +scalariformFormat +test:scalariformFormat
+      - name: Check code is formatted
+        run: sbt scalafmtCheckAll scalafmtSbtCheck headerCheckAll
```

Comment:
So this is just temporary, i.e. rather than doing an auto format we just check that the code is formatted.

This will be replaced in the future with the scalafmt-native github action